api/views/user_views.py

```diff
@@ -213,6 +220,41 @@ def post(self, request):
         return JsonResponse(response, status=status.HTTP_200_OK)
 
 
+@extend_schema(tags=["Authentication Endpoint"])
+class JWTLoginView(APIView):
```

Comment:
I don't believe we have default throttling set up; this (and a few other) endpoints need to be throttled; perhaps we can add at least one Jira issues for that.

Reply:
That's probably for the best so we can add more details on the specific limits and discuss them beforehand.